mcs/class/System.Net.Http/HttpClientHandler.SocketsHandler.Android.cs

```diff
@@ -6,7 +8,14 @@ static IMonoHttpClientHandler CreateDefaultHandler ()
 		{
 			string envvar = Environment.GetEnvironmentVariable ("XA_HTTP_CLIENT_HANDLER_TYPE")?.Trim ();
 			if (envvar?.StartsWith("System.Net.Http.MonoWebRequestHandler", StringComparison.InvariantCulture) == true)
-				return new MonoWebRequestHandler ();
+			{
+				Type monoWrhType = Type.GetType (envvar, false);
+				if (monoWrhType != null)
+					return (IMonoHttpClientHandler) Activator.CreateInstance (monoWrhType);
+				else
+					Console.WriteLine ($"{envvar} type was not found.");
```

Comment:
Please no Console (it adds dependency)

Reply:
@marek-safar it's been there for quite a while, see https://github.com/mono/mono/blob/master/mcs/class/System.Net.Http/System.Net.Http/HttpClient.android.cs#L40 I just copied it. Can remove it from this assembly

Reply:
Also, isn't Console part of corelib (for classic mono?)

Reply:
Removed